### PR TITLE
AKS Baseline regulated - AKS 1.23.3 version bump - Also fixing jettech/kube-webhook-certgen bug in the docs and also increased app gtwy subnet size

### PR DESCRIPTION
**Pull request by magrande**

### CODE REVIEW

File: docs/deploy/10-pre-bootstrap.md [hunk 1]
```diff
@@ -53,9 +53,9 @@ Using a security agent that is container-aware and can operate from within the c
    az acr import --source ghcr.io/fluxcd/source-controller:v0.8.1 -t quarantine/fluxcd/source-controller:v0.8.1 -n $ACR_NAME_QUARANTINE       && \
    az acr import --source docker.io/falcosecurity/falco:0.29.1 -t quarantine/falcosecurity/falco:0.29.1 -n $ACR_NAME_QUARANTINE               && \
    az acr import --source docker.io/library/busybox:1.33.0 -t quarantine/library/busybox:1.33.0 -n $ACR_NAME_QUARANTINE                       && \
-   az acr import --source docker.io/weaveworks/kured:1.9.0 -t quarantine/weaveworks/kured:1.9.0 -n $ACR_NAME_QUARANTINE                       && \
+   az acr import --source docker.io/weaveworks/kured:1.9.2 -t quarantine/weaveworks/kured:1.9.2 -n $ACR_NAME_QUARANTINE                       && \
    az acr import --source k8s.gcr.io/ingress-nginx/controller:v1.1.2 -t quarantine/ingress-nginx/controller:v1.1.2 -n $ACR_NAME_QUARANTINE    && \
-   az acr import --source docker.io/jettech/kube-webhook-certgen:v1.1.1 -t quarantine/jettech/kube-webhook-certgen:v1.1.1 -n $ACR_NAME_QUARANTINE
+   az acr import --source k8s.gcr.io/ingress-nginx/kube-webhook-certgen:v1.1.1 -t quarantine/jettech/kube-webhook-certgen:v1.1.1 -n $ACR_NAME_QUARANTINE
```
Comment (magrande) [inline docs/deploy/10-pre-bootstrap.md:58]: This is a bug I introduced. The image path is wrong currently

File: cluster-manifests/ingress-nginx/deployment.yaml [hunk 2]
```diff
@@ -22,7 +22,7 @@ data:
   #https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/
   # This is our Azure Application Gateway range. The ONLY traffic that should be hitting
   # the ingress controller directly is AAG.
-  whitelist-source-range: 10.240.4.16/28
+  whitelist-source-range: 10.240.4.0/24
```
Comment (ckittel) [inline cluster-manifests/ingress-nginx/deployment.yaml:25]: This will be 5.0/24

File: cluster-stamp.v2.json [hunk 3]
```diff
@@ -662,7 +662,7 @@
                             "backendAddresses": [
                                 {
                                     /* This is the IP address that our ingress controller will request */
-                                    "ipAddress": "10.240.4.4"
+                                    "ipAddress": "10.240.3.244"
```
Comment (ckittel) [inline cluster-stamp.v2.json:665]: This will also be unchanged

File: networking/spoke-BU0001A0005-01.json [hunk 4]
```diff
@@ -762,7 +762,7 @@
                     {
                         "name": "snet-applicationgateway",
                         "properties": {
-                            "addressPrefix": "10.240.4.16/28",
+                            "addressPrefix": "10.240.4.0/24",
```
Comment (ckittel) [inline networking/spoke-BU0001A0005-01.json:765]: Move to 5.0/24

**Review by ckittel — CHANGES_REQUESTED** [review]

File: cluster-manifests/ingress-nginx/deployment.yaml [hunk 5]
```diff
@@ -239,7 +239,7 @@ metadata:
     service.beta.kubernetes.io/azure-load-balancer-internal-subnet: snet-cluster-ingressservices
 spec:
   type: LoadBalancer
-  loadBalancerIP: 10.240.4.4
+  loadBalancerIP: 10.240.3.244
```
[the remark this answers is not in the document]
Comment (magrande) [reply cluster-manifests/ingress-nginx/deployment.yaml]: Done

File: cluster-manifests/ingress-nginx/deployment.yaml (hunk 2 above)
Comment (magrande) [reply cluster-manifests/ingress-nginx/deployment.yaml]: Done

File: cluster-stamp.v2.json (hunk 3 above)
Comment (magrande) [reply cluster-stamp.v2.json]: Done

File: networking/spoke-BU0001A0005-01.json [hunk 6]
```diff
@@ -751,7 +751,7 @@
                     {
                         "name": "snet-cluster-ingressservices",
                         "properties": {
-                            "addressPrefix": "10.240.4.0/28",
+                            "addressPrefix": "10.240.3.240/28",
```
[the remark this answers is not in the document]
Comment (magrande) [reply networking/spoke-BU0001A0005-01.json]: Reverted

**Review by ckittel — APPROVED** [review]
LGTM!

**Pull request merged**